backend/BackendMIIT/src/main/resources/application.properties

```diff
@@ -1,6 +1,7 @@
 spring.application.name=BackendMIIT
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create
 spring.datasource.url=jdbc:postgresql://localhost:5432/MiitInside
 spring.datasource.username=postgres
 spring.datasource.password=postgres
 spring.datasource.driver-class-name=org.postgresql.Driver
+webclient.baseurl="https://localhost:8080"
```

Comment:
Add the empty line at the end of the file

Reply:
Added